it/content/src/main/content/jcr_root/conf/core-components-it/settings/wcm/policies/.content.xml

```diff
@@ -12,7 +12,7 @@
                       jcr:primaryType="nt:unstructured"
                       jcr:title="formcontainer-policy"
                       sling:resourceType="wcm/core/components/policy/policy"
-                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker]">
+                      components="[/apps/forms-components-examples/components/form/textinput,/apps/forms-components-examples/components/form/datepicker,/libs/core/fd/components/form/radiobutton/v1/radiobutton]">
```

Comment:
It should be the path of /apps/forms-component-examples and not libs

Reply:
done